operator/Makefile

```diff
@@ -115,16 +115,24 @@ generate: $(CONTROLLER_GEN) ## Generate controller and crd code
 manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
-test: deps generate go-generate lint manifests ## Run tests
+test: deps generate go-generate lint lint-prom manifests ## Run tests
 test: $(GO_FILES)
 	go test ./... -coverprofile cover.out
 
+.PHONY: test-unit-prom
+test-unit-prom: $(PROMTOOL) ## Run prometheus unit tests
```

Comment:
Does it make sense to spell out `prometheus` or is that too inconvenient? (same for other target)